MapboxNavigation/RouteMapViewController.swift

```diff
@@ -808,18 +809,56 @@ extension RouteMapViewController: NavigationViewDelegate {
                     } else {
                         currentName = nil
                     }
+                    
+                    if let line = feature as? MGLPolylineFeature {
```

Comment:
We've already pulled this value out above. See if you can combine these if let statements.